media/css/firefox/sync.less

```diff
@@ -133,6 +137,18 @@
     }
 }
 
+.primary-buttons {
+    display: flex;
+    justify-content: space-around;
```

Comment:
Happy to synchronize Swatches (did you have that show in the UK?). Should we get #3765 merged first, then I can rebase and convert?